src/utils/testing/mocks/mockNote.ts

```diff
@@ -0,0 +1,20 @@
+import { mockObject } from 'utils/testing/mocks';
+import { ZetkinNote } from 'types/zetkin';
+
+const note: ZetkinNote = {
+  id: 1,
+  text: JSON.stringify([
+    {
+      children: [{ text: 'Note to self' }],
+      type: 'paragraph',
+    },
+  ]),
```

Comment:
This is not correct after changing to markdown, right?

Reply:
I think this should change or it will confuse us later.

Reply:
Good spot - I hadn't noticed this